org.adempiere.base/src/org/compiere/model/MIssue.java

```diff
@@ -53,10 +53,10 @@
 	 */
 	public static MIssue create (LogRecord record)
 	{
-		s_log.config(record.getMessage());
+		if (s_log.isLoggable(Level.CONFIG))
+			s_log.config(record.getMessage());
 		MSystem system = MSystem.get(Env.getCtx()); 
-		if (!DB.isConnected() 

```

Comment:
Hi @hengsin - I don't understand why to drop the validation if the DB is not connected, I think it is there because sometimes an Issue is created before the DB plugin is started, or because of database failure, and in such case trying to save an Issue will create a new Issue and probably it will end in a recursive endless loop there.

Reply:
that's actually redundant as DB.isConnected check is already done in the CLogErrorBuffer.publish method. anyway, I've add it back since it is harmless to have it there as extra guard.